streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -74,10 +74,12 @@ class FileInputDStream[K, V, F <: NewInputFormat[K,V]](
     directory: String,
     filter: Path => Boolean = FileInputDStream.defaultFilter,
     newFilesOnly: Boolean = true,
-    conf: Option[Configuration] = None)
+    @transient conf: Option[Configuration] = None)
```

Comment:
I think that's unnecessary. If the `Configuration` is `private val` and isn't used except in the constructor statements, it's not even a field right? so doesn't even become something to serialize.